### ISSUE TITLE
Creating a hidden group with all of the objects can sometimes cause a 'Group not found' error

### ISSUE THREAD

**Issue by The-Daniel** [3drepo/3drepo.io#4724]
### Description

Seems to only occur when the number of hidden objects is large (8000 in my case). Creating a ticket group and setting all of the objects to be hidden causes an error modal to appear stating that the group id was not found. You are then redirected to the dashboard. Upon returning the group was created successfully.

- This is happening in: both 
- This requires a hotfix to production: no 

### Steps to replicate
- Open a model with a large number of objects (e.g. 8000cubes which can be found on sharepoint)
- Create a group ticket. Selecting all of the objects and hiding them
- Save the group

### Current Behaviour
- An error modal appears which send the user to the dashboard

### Expected Behaviour
- No error modal of redirect occurs

### Pictures/ Gifs of the error
![image](https://github.com/3drepo/3drepo.io/assets/62481250/12173d6b-e92e-4041-9db5-d0b44d1f9180)

**Comment by carmenfan** [comment]
This is cased by a race condition on the backend. 

The event is new ticket is triggered when we've saved the ticket into the database https://github.com/3drepo/3drepo.io/blob/master/backend/src/v5/models/tickets.js#L42

Groups are saved as external data after the ticket is created (because we need to know the Ticket ID before we can insert something https://github.com/3drepo/3drepo.io/blob/master/backend/src/v5/processors/teamspaces/projects/models/commons/tickets.js#L166

So In the event when the group is large thus taking more time to save. The event could've passed the message to the front end, and front end would've triggered a request to get the group prior to the group is actually saved.

So the fix will be to only trigger the ticket event after we know the full ticket data has been created

**Comment by carmenfan** [comment]
Closing, I'm rewriting this for #4777 so will fix it in there.
